docs/core/basics_en.md

```diff
@@ -1,6 +1,6 @@
 # Introduction & Basics
 
-A dogu (derived from Japanese for "tool") provides a containerized application in the Cloudogu EcoSystem (CES).
+A dogu (derived from Japanese for "tool") provides a containerized application in the [Cloudogu EcoSystem (CES)](https://docs.cloudogu.com/en/docs/introduction/).
 Dogus can satisfy different usage aspects, such as a project management (towards users) or a database management system (towards other dogus).
 It follows that dogus must be able to do more than would be possible with pure Docker directives. Thinking of the above dogus, it makes sense, among other things, to have a ServiceAccount mechanism for communication from
```

Comment:
Sounds not quite right to me without the additional `what` but i may be wrong here.
```suggestion
It follows that dogus must be able to do more than what would be possible with pure Docker directives. Thinking of the above dogus, it makes sense, among other things, to have a ServiceAccount mechanism for communication from
```

Reply:
Mhh.. For me it sounds wrong with the `what`..
Let us trust deepl :smile: